strings/find_anagram_indices.py

```diff
@@ -0,0 +1,81 @@
+from collections import Counter, defaultdict
+
+
+def find_anagram_indices(word: str, letters: str) -> list:
+    """
+    Given a word `word`, and a string `letters`,
+    find all indices in `letters` which are the starting
+    locations of anagrams of `word`.
+    For example, given `word` is `the` and
+    `letters` is `tehxetheht`, return `[0, 4, 5, 7]`.
+
+    1. Brute force solution
+
+    This algorithm would take O(w x s) time
+    - x is the length of the word
+    - y is the length of the input string
+
+    :param word: word to be analyzed
+    :param letters: complete word to be scanned
+    :return: Array
```

Comment:
`:return:` should be filled with what the function returns in simple English as you did with `:param:`. If you want to specify the return type, the word is `:rtype:` which is not needed here as we are annotating the function directly.